code/modules/reagents/dispenser/dispenser_presets.dm

```diff
@@ -118,6 +118,22 @@
 			/obj/item/weapon/reagent_containers/chem_disp_cartridge/mead
 		)
 
+/obj/machinery/chemical_dispenser/coffeemaster

```

Comment:
It would be nice if we could get some custom sprites for that.
Maybe ask in #code_dungeon if someone is willing.

Reply:
Made a sprite just now.